Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -52,6 +53,20 @@
 
 #include "UICommon/GameFile.h"
 
+#if defined(_WIN32)
+static bool FFmpegWin32Exists()
```

Comment:
This should probably go in ``UICommon`` somewhere

Reply:
Would this require including `UICommon.h` in the `MenuBar.cpp` if I put it in UICommon.cpp for example? You'd be okay with that?